pyproject.toml

```
@@ -0,0 +1,38 @@
+[tool.poetry]
+name = "epcsunspecdemo"
+version = "0.0.0"
+description = ""
```

```suggestion
description = "EPC SunSpec demonstration tool"
```
This is the description from the now removed `setup.py`.